.env.development

```diff
@@ -7,7 +9,10 @@ VITE_KC_SITE_BASE_URL=https://dev.zoo.dev
 VITE_KC_SITE_APP_URL=https://app.dev.zoo.dev
 VITE_KC_SKIP_AUTH=false
 VITE_KC_CONNECTION_TIMEOUT_MS=5000
-# ONLY add your token in .env.development.local if you want to skip auth, otherwise this token takes precedence!
-#VITE_KC_DEV_TOKEN="your token from dev.zoo.dev should go in .env.development.local"
+#VITE_KC_DEV_TOKEN="optional token from dev.zoo.dev to skip auth in the app"
+
+RUST_BACKTRACE="full"
+PYO3_PYTHON=/usr/local/bin/python3
+#KITTYCAD_API_TOKEN="required token from dev.zoo.dev for engine testing"
```

Comment:
Do the Rust tools read this file?

Reply:
What I'm trying to say is, I don't think any of the Rust tools read this file or the `.local` one. So by removing `RUST_BACKTRACE="full"` from the `justfile` and adding it here, I don't think the functionality is preserved. I don't personally use `just test`, so it doesn't really bother me. I'm going to use my own environment variables anyway. (I've been using `direnv` for times when I don't control the invocation.) But maybe you want to know.

Reply:
They don't explicitly need this file. This is merely a convenient way of enumerating and setting environment variables.